src/components/AppNavigation/CalendarList/CalendarListItem.vue

```diff
@@ -342,6 +335,17 @@ export default {
 
 			return ''
 		},
+	  /**
+	   * compute aria-description for AppNavigationItem link
+	   *
+	   * @return {string}
+	   */
+	  descriptionAppNavigationItem() {
+		  const untitledCalendar = t('calendar', '{calendar}', { calendar: 'Untitled calendar' })
```

Comment:
I'm not sure what you're doing there, but the translation for 'Untitled calendar' won't be used that way.

In any case, it's hard to assume the order of the words will stay the same, it's possible there's a language where the position of the calendar name depends if the calendar has a real name or not. Imagine a situation like :
* `Disable calendar "MyCalendar"` :arrow_right: `"Disable calendar "MyCalendar"` :heavy_check_mark: 
* `Disable calendar "Untitled Calendar"` :arrow_right: `"Untitled Calendar" Disable calendar` :no_entry_sign: 

It's better to have all 4 cases, even though it's long:
* `Disable calendar "{calendar}"`
* `Disable untitled calendar`
* `Enable calendar "{calendar}"`
* `Enable untitled calendar`